sw/device/silicon_creator/lib/BUILD

```diff
@@ -223,6 +223,16 @@ dual_cc_library(
     ),
 )
 
+dual_cc_library(
+    name = "manifest_def",
+    srcs = dual_inputs(device=["manifest_def.c"]),
+    hdrs = dual_inputs(shared=["manifest_def.h"]),
```

Comment:
If you're using only `shared`, you can just use a list like `hdrs = ['mainfest_def.h']`